### ISSUE TITLE
CVE record and security advisory are out of sync

### ISSUE THREAD

**Issue by mlaponder** [rollup/rollup#5702]
### Rollup Version

2.79.2

### Operating System (or Browser)

Any

### Node Version (if applicable)

NA

### Link To Reproduction

NA

### Expected Behaviour

I got a security issue reported by Snyk (https://app.snyk.io/vuln/SNYK-JS-ROLLUP-8073097) and it refers to https://www.cve.org/CVERecord?id=CVE-2024-47068. 
Both are mentioning patch versions 3.29.5 and 4.22.4. But when I check the rollup security advisory https://github.com/rollup/rollup/security/advisories/GHSA-gcx4-mw62-g8wm I see also a patch for 2.79.2.

I would expect the CVE and your security advisory be in sync. Is my understanding correct this issue has also been patched to 2.79.2, and if so how can this get reflected in the CVE (and eventually snyk)?

### Actual Behaviour

CVE & Snyk are out of sync on the version patch 2.79.2

**Comment by lukastaegert** [comment]
Yes. The problem is that the patch for 2.79.2 was published some time after the CVE had already been published and I am not sure how I can edit a CVE.

**Comment by mlaponder** [comment]
@lukastaegert I have no experience with that as well, but afaics you would need to request an update of the CVE here https://www.cve.org/ReportRequest/ReportRequestForNonCNAs#UpdateCVERecord. I am also willing to try it, but I am not rollup maintainer (not sure if that would be an issue).

**Comment by lukastaegert** [comment]
I wrote GitHub a message, let's see if they update the CVE.

**Comment by sterliakov** [comment]
All the links above now mention 2.79.2 as patched - can this issue be closed now?
